### DISCUSSION TITLE
如何正确的升级nonebot2

### DISCUSSION THREAD

**Discussion by fatinghenji** [nonebot/discussions#38] [问答] [answered]
今天安装插件的时候，提示`elf-rss 2.4.2 requires nonebot2==2.0.0a15, but you have nonebot2 2.0.0a13.post1 which is incompatible. `
请问应该如何正确的升级nonebot2呢？

**Comment by yanyongyu** [comment, marked as answer]
> 请问应该如何正确的升级nonebot2呢？

`pip install --upgrade nb-cli nonebot2 nonebot-adapter-cqhttp`
